scheduler/docs/configuration.adoc

```diff
@@ -89,13 +89,12 @@ If need it to bind to another port, you can specify that with the `:local-port`
 `:sandbox-syncer`::
```

Comment:
I think the name `sandbox-syncer` isn't appropriate now that it's just a publisher.

Reply:
Renamed to `sandbox-publisher`